Sources/LanguageServerProtocol/Request.swift

```diff
@@ -123,16 +106,24 @@ extension Request: CustomStringConvertible, CustomLogStringConvertible {
   }
 }
 
-extension Notification: CustomStringConvertible, CustomLogStringConvertible {
+fileprivate struct AnyNotificationType: CustomLogStringConvertible {
```

Comment:
Would be good to define description/redactedDescription outside of LanguageServerProtocol (ie. remove the LSPLogging dependency)

Reply:
Filed https://github.com/apple/sourcekit-lsp/issues/936